### ISSUE TITLE
Transaction show as pending on Rwallet but was already confirmed

### ISSUE THREAD

**Issue by bitcoinuser** [rsksmart/rwallet#451]
I did a Swap on rskswap.com using Nifty. (I have the same seed on Nifty and on Rwallet).
For some reason on Rwallet the transaction show as pending for more than 11 hours but on explorer and on Nifty is confirmed.

https://explorer.rsk.co/tx/0x5e60b055087e2609f6442f3e42c36dbb88084faec2af6afc559fc30e35650a1a

![photo_2020-08-31_22-54-09](https://user-images.githubusercontent.com/1178856/91785969-e8d7a300-ebdc-11ea-81b2-19fe3cd0aa74.jpg)

**Comment by chrisli30** [comment]
Thanks for reporting. @imstar15 please take a look at that transaction.

**Comment by bitcoinuser** [comment]
If you could give priority to this bug, I appreciate. This make impossible to me use Rwallet now. Thanks.

**Comment by chrisli30** [comment]
@bitcoinuser yes we will. @tinyyxx  please take a look at this issue from server side today.

**Comment by bitcoinuser** [comment]
News?

**Comment by imstar15** [comment]
The status field value in the back-end database is 0-pending, but the status of explorer is successful. I think there is a problem with the data synchronization between the backend and the chain, and @tinyyxx  needs to find the specific reason and modify it.

**Comment by bitcoinuser** [comment]
Today I had this problem again. I sent a rbtc transaction, the transaction was confirmed here: 0xddda27671996d8c4c6b0b3127b28f66afd6bccda3db910bdca34a4c8f5c00582
But on RWallet appear as Sending: 

![photo_2020-11-06_19-10-31](https://user-images.githubusercontent.com/1178856/98419417-caba7300-2063-11eb-88ee-6a50632bc0be.jpg)

**Comment by chrisli30** [comment]
@tinyyxx please check the status of that transaction in the backend and the cause of it.

**Comment by tinyyxx** [comment]
This might be rskExploreApi didn't push successful tx data to our server.
I've added more detailed log to track the this issues.
